### ISSUE TITLE
Many <color> in value, `scrollbar-color` for example

### ISSUE THREAD

**Issue by ariarzer** [AndyOGo/stylelint-declaration-strict-value#117]
I use this config:
```
scale-unlimited/declaration-strict-value': [
  [ '/color/', 'fill', 'stroke' ],
  {
    ignoreKeywords: [ 'currentColor', 'transparent', 'inherit', 'unset', ],
  }],
```

When i use a 
```css
scrollbar-color: var(--steel_gray_140) transparent;
```
I get an error 
```
Expected variable, function or keyword for "var(--steel_gray_140) transparent" of "scrollbar-color"
```

This [property](https://drafts.csswg.org/css-scrollbars-1/#scrollbar-color) doesn't have a shorthand, but use a `<color>{2}` grammar. But the plugin tries to check the entire property value.

**Comment by AndyOGo** [comment]
@ariarzer Thank you for the issue.

Fixing this properly requires CSS value parsing, for this purpose this module utilizes `css-values`, which does not support `scrollbar-color` yet.

You can find a full list of supported CSS values here:
https://github.com/ben-eb/css-values#property-support

**Comment by AndyOGo** [comment]
@ariarzer 
I thought about your issue and multi-value grammars in general.
For this case I decided to just split CSS values and avoid parsing.

It's released with [`v1.7.0`](https://github.com/AndyOGo/stylelint-declaration-strict-value/blob/master/CHANGELOG.md#170-2020-11-19).
This should improve any multi-value properties (excluding shorthand).
